File: oeml-sdk/javascript-closure-angular/API/Client/Fills.js
goog.provide('API.Client.Fills');

/**
 * @record
 */
API.Client.Fills = function() {}

/**
 * Execution time.
 * @type {!API.Client.date}
 * @export
 */
API.Client.Fills.prototype.time;

/**
 * Execution price.
 * @type {!number}
 * @export
 */
API.Client.Fills.prototype.price;

/**
 * Executed quantity.
 * @type {!number}
 * @export
 */
API.Client.Fills.prototype.amount;

File: oeml-sdk/dart-jaguar.yaml/lib/model/fills.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'fills.jser.dart';

class Fills {
   /* Execution time. */
  @Alias('time', isNullable: false,  )
  final DateTime time;
   /* Execution price. */
  @Alias('price', isNullable: false,  )
  final num price;
   /* Executed quantity. */
  @Alias('amount', isNullable: false,  )
  final num amount;
  

  Fills(
      

{
     this.time = null,  
     this.price = null,  
     this.amount = null 
    
    }
  );

  @override
  String toString() {
    return 'Fills[time=$time, price=$price, amount=$amount, ]';
  }
}

@GenSerializer(nullableFields: true)
class FillsSerializer extends Serializer<Fills> with _$FillsSerializer {

}

